backend/server/Cargo.toml

```diff
@@ -15,9 +15,11 @@ figment = { version = "0.10.19", features = ["env", "toml"] }
 http = "1.3.1"
 jsonwebtoken = "9.3.0"
 octocrab = "0.41.2"
+once_cell = "1.21.3"
 serde = { workspace = true }
 serde_json = { workspace = true }
 shared = { workspace = true }
+sqlx = { version = "0.8.3", features = [ "runtime-tokio", "tls-native-tls", "sqlite" ] }
```

Comment:
Checking https://docs.rs/crate/sqlx/latest/features, I think we should go with `no-default-features = true`, because we do not need `any` and `json` at the moment (`json` we might need in the future). Also I think we do not actually need any tls feature, since SQlite connections are all local anyways.

Reply:
done